src/table/components/TableHeadWrapper.tsx

```diff
@@ -89,7 +79,31 @@ function TableHeadWrapper({
                   </VisuallyHidden>
                 )}
               </StyledSortLabel>
-            </TableCell>
+            );
+          };
+
+          return (
+            <StyledTableCell
+              sx={{ ...headerStyle, zIndex: columns.length - columnIndex }}
+              key={column.id}
+              align={column.align}
+              className="sn-table-head-cell sn-table-cell"
+              tabIndex={tabIndex}
+              aria-sort={ariaSort}
+              aria-pressed={isCurrentColumnActive}
+              onKeyDown={handleKeyDown}
+              onMouseDown={() => handleClickToFocusHead(columnIndex, rootElement, setFocusedCellCoord, keyboard)}
+              onClick={(evt: React.MouseEvent) => handleClickToSort(evt, column, changeSortOrder, isInteractionEnabled)}
+            >
+              {areBasicFeaturesEnabled ? (
+                <HeadCellContent>
```

Comment:
wait, why are you only using `HeadCellContent` when the flag is on? I think it could be there for both cases. Tried it quickly and it looks better when the arrows are next to the text, more like the design. Will save some work when I fix the styling later

Reply:
What do you mean? when the flag is on it shows everything as the design says.  When it is off it looks exactly as same as it is on the old table. shouldn't it be like that  with flag

Reply:
I guess you might forgot to set the flag to `true` when you run it on nebula ?